spec/operations/operation_needs_spec.cr

```diff
@@ -72,12 +82,15 @@ describe "Avram::SaveOperation needs" do
   end
 
   it "also generates named args for other attributes" do
-    NeedsSaveOperation.create(name: "Jane", nilable_value: "not nil", optional: "bar", created_by: "Jane") do |operation, _record|
+    uploaded_file = Avram::UploadedFile.new("thumb.png")
+    NeedsSaveOperation.create(name: "Jane", nilable_value: "not nil", optional: "bar", created_by: "Jane", not_db_related: 4, image: uploaded_file) do |operation, _record|
       # Problem seems to be that params override passed in val
       operation.name.value.should eq("Jane")
       operation.nilable_value.should eq("not nil")
       operation.created_by.should eq("Jane")
       operation.optional.should eq("bar")
+      operation.not_db_related.value.should eq(4)
```

Comment:
this is not necessarily related, but I added it in just to ensure nothing got messed up with `attribute` and `file_attribute` being added in together.